packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTreeComponent.tsx

```diff
@@ -69,106 +67,184 @@ export const ModelsTreeComponent = (props: ModelTreeProps) => {
       });
   }, [queryModels, viewport]);
 
-  const invert = useCallback(async () => {
-    if (availableModels.length === 0) return;
-    const notViewedModels: string[] = [];
-    const models: string[] = [];
-    availableModels.forEach((id: string) => {
-      if (viewport?.viewsModel(id)) models.push(id);
-      else notViewedModels.push(id);
-    });
-    await viewport?.addViewedModels(notViewedModels);
-    viewport?.changeModelDisplay(models, false);
-    viewport?.invalidateScene();
-  }, [viewport, availableModels]);
+  return (
+    <>
+      {iModel && viewport &&
+        <>
+          <TreeHeaderComponent
+            searchOptions={searchOptions}
+            treeHeaderButtons={props.TreeHeaderButtons
+              ? props.TreeHeaderButtons.map((btn) => btn({ viewport, iModel, availableModels }))
```

Comment:
React logs an error in console for missing `key` property.

Reply:
fixed